sass/edx/_variables.scss

```diff
@@ -9,15 +9,15 @@
 //
 
 $white:  #fff !default;
-$gray-100: #f8f9fa !default;
-$gray-200: #e9ecef !default;
-$gray-300: #dee2e6 !default;
-$gray-400: #ced4da !default;
-$gray-500: #adb5bd !default;
-$gray-600: #868e96 !default;
-$gray-700: #495057 !default;
-$gray-800: #343a40 !default;
-$gray-900: #212529 !default;
+$gray-100: #f5f5f5 !default;
```

Comment:
Reviewers: I mostly took these grays from the Pattern Library spec:

http://ux.edx.org/design_elements/colors/